### PR TITLE
TR person search v3

### LINKED ISSUE #12: Switch to TR API v3 for person search

### PR DESCRIPTION
**Pull request by jc00ke**
Closes #12 

Tested locally in `iex` and in the UI, everything works.

### CODE REVIEW

File: lib/epi_locator_web/templates/tr/person_search.xml.eex [hunk 1]
```diff
@@ -39,7 +39,7 @@
     </Criteria>
     <Datasources>
         <PublicRecordPeople>true</PublicRecordPeople>
-        <NPIRecord>true</NPIRecord>
-        <WorldCheckRiskIntelligence>true</WorldCheckRiskIntelligence>
+        <NPIRecord>false</NPIRecord>
```
Comment (jc00ke) [inline lib/epi_locator_web/templates/tr/person_search.xml.eex:42]: Undocumented change from v2 to v3. v2 would silently ignore these if they were disabled based on the products paid for by customer. v3 will return an error, so I'm specifically disabling.

File: lib/epi_locator/tr/tr_client.ex [hunk 2]
```diff
@@ -61,7 +61,7 @@ defmodule EpiLocator.TRClient do
   end
 
   @impl EpiLocator.TRClientBehaviour
-  def person_search_results(uri), do: search_results(uri, "{http://clear.thomsonreuters.com/api/search/2.0}PersonResultsPage")
+  def person_search_results(uri), do: search_results(uri, "{http://clear.thomsonreuters.com/api/search/2.0}PersonResultsPageV3")
```
Comment (patmaddox) [inline lib/epi_locator/tr/tr_client.ex:64]: What's this syntax with the curly brackets about? Clearly it's been there for a while, it just seems weird to me.

File: lib/epi_locator_web/templates/tr/person_search.xml.eex [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<ps:PersonSearchRequest xmlns:ps="http://clear.thomsonreuters.com/api/search/2.0">
+<ps:PersonSearchRequestV3 xmlns:ps="http://clear.thomsonreuters.com/api/search/2.0">
```
Comment (patmaddox) [inline lib/epi_locator_web/templates/tr/person_search.xml.eex:2]: So it's `PersonSearchRequestV3` but still under `http://clear.thomsonreuters.com/api/search/2.0` ? (V3 vs 2.0)

File: test/support/helpers.ex [hunk 4]
```diff
@@ -10,6 +10,6 @@ defmodule EpiLocator.Test.Helpers do
   def read_tr_xml_and_turn_into_map(filename) do
     filename
     |> read_xml_and_turn_into_map()
-    |> Map.get("{http://clear.thomsonreuters.com/api/search/2.0}PersonResultsPage")
+    |> Map.get("{http://clear.thomsonreuters.com/api/search/2.0}PersonResultsPageV3")
```
Comment (patmaddox) [inline test/support/helpers.ex:13]: `/api/v3` ... `/api/search/2.0` ... cool

(kind of a weird naming / versioning scheme they've got going on imo)

**Review by patmaddox — APPROVED** [review]

File: lib/epi_locator/tr/tr_client.ex (hunk 2 above)
Comment (jc00ke) [reply lib/epi_locator/tr/tr_client.ex:64]: It's some XML thing. I'm not sure what it's called.

File: lib/epi_locator_web/templates/tr/person_search.xml.eex (hunk 3 above)
Comment (jc00ke) [reply lib/epi_locator_web/templates/tr/person_search.xml.eex:2]: You are correct. `PersonSearchRequestV3` but still under `2.0` namespace. :man_shrugging:

**Pull request merged**